siddhant/random-wallpaper-wallhaven.py

```diff
@@ -0,0 +1,31 @@
+import requests, bs4, os, random
+#import subprocess
```

Comment:
A few nits:
1. Remove all commented out code.
2. Use either single quotes or double quotes for strings. (Prefer single quotes)

Reply:
Commented out code will be removed.
Will change accordingly, btw. any reason for preferring single quotes :stuck_out_tongue_winking_eye:

Reply:
Great, I (now) tend to follow the Google Python Style guide. For strings, see [this](https://github.com/google/styleguide/blob/gh-pages/pyguide.md#310-strings).

Reply:
Thanks! Will follow it.